gno.land/pkg/gnoland/param.go

```diff
@@ -0,0 +1,121 @@
+package gnoland
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/gnolang/gno/tm2/pkg/sdk"
+	"github.com/gnolang/gno/tm2/pkg/sdk/params"
+)
+
+type Param struct {
+	key   string
+	kind  string
```

Comment:
Just curious, why not define an official `ParamKind` type and use that explicitly?

Reply:
As an enum or just as `type ParamKind string`?

Reply:
Just as a `type ParamKind string`

I don't see it having any benefit apart from readability